tools/operator/launcher/networkspec.yaml

```diff
@@ -5,7 +5,7 @@
 #! fabricVersion:
 #! Released images are pulled from docker hub hyperledger/, e.g. 1.4.1 or 2.0.0
 #! Development stream images are pulled from
-#! nexus3.hyperledger.org:10001/hyperledger/, e.g. 1.4.1-stable or 2.0.0-stable
+#! hyperledger-fabric.jfrog.io/, e.g. 1.4.1-stable or 2.0.0-stable
 fabricVersion: 1.4.5-stable
```

Comment:
Are we publishing images with this tag?

Reply:
Which tag?

Reply:
1.4.5-stable?

Reply:
I'll update the example to say 1.4.5-stable, but that was just an example